runebender-lib/src/data.rs

```diff
@@ -356,6 +358,33 @@ impl Workspace {
         }
     }
 
+    pub fn set_glyph_group(&mut self, prefix: &str, glyph_name: GlyphName, group_name: String) {
```

Comment:
This definitely reenforces my suspicion that we should find some better way to store these. 🤔

Reply:
I agree, though it's hard to think of a good clean place to introduce yet another intermediate datastructure